app/api/models/challenge-request.ts

```diff
@@ -0,0 +1,54 @@
+/* tslint:disable */
+/* eslint-disable */
+/**
+ * API
+ * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+ *
+ * The version of the OpenAPI document: 1
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+
+
+/**
+ * 
+ * @export
+ * @interface ChallengeRequest
+ */
+export interface ChallengeRequest {
+    /**
+     * 
+     * @type {string}
+     * @memberof ChallengeRequest
+     */
+    'title'?: string;
+    /**
```

Comment:
please check it. there is title?: string, it should be title: string, just add adnotation to a request in java

Reply:
for all fields